tests/load/locust_tests/locustfile.py

```diff
@@ -305,6 +372,27 @@ def top_picks_suggestions(self) -> None:
         for query in queries:
             request_suggestions(self.client, query, providers)
 
+    @task(weight=1)
+    def weather_suggestions(self) -> None:
+        """Send multiple requests for Weather queries."""
+        full_query: str = choice(WEATHER_QUERIES)  # nosec
+        providers: str = "weather"
```

Comment:
we haven't renamed the provider yet.
```suggestion
        providers: str = "accuweather"
```